Makefile

```diff
@@ -109,6 +109,10 @@ bumpver: docs
 	make -C po pykickstart.pot ; \
 	zanata push $(ZANATA_PUSH_ARGS)
 
+pykickstart.spec:
+	cp pykickstart.spec.in pykickstart.spec
+	sed -i "s/%%VERSION%%/$(VERSION)/" pykickstart.spec
```

Comment:
Style concern here.  Can you do this in one line instead, like:

sed -e "s/%%VERSION%%/$(VERSION)/" < $< > $@

Changing the target so it lists pykickstart.spec.in as a requirement

Reply:
This can be solved also by the command `sed "s/%%VERSION%%/$(VERSION)/" $< > $@`.
But I will use your approach.